extensions/replication/tasks/UpdateReplicationStatus.js

```diff
@@ -277,7 +371,7 @@ class UpdateReplicationStatus extends BackbeatTask {
             const params = { sourceEntry, refreshedEntry };
             const updatedSourceEntry = this._getUpdatedSourceEntry(params, log);
             if (!updatedSourceEntry) {
-                return done();
+                return process.nextTick(done);
```

Comment:
Original code had this 'nextTick()', but not sure it is still relevant in the updated code

@nicolas2bert Can you provide more context/feedback?

Reply:
I will keep it to keep the callback function asynchronous in both cases.

Reply:
thanks